packages/cli/src/main/kotlin/elide/tool/cli/output/Mosaic.kt

```diff
@@ -183,14 +183,14 @@ private fun Summary(start: Long, totalTests: Int, tests: List<Test>) {
   val passed = counts[Pass] ?: 0
   val running = counts[Running] ?: 0
 
-  var elapsed by remember { mutableStateOf(0) }
+  // var elapsed by remember { mutableStateOf(0) }
 
   LaunchedEffect(Unit) {
     while (true) {
       delay(1_000)
-      Snapshot.withMutableSnapshot {
-        elapsed++
-      }
+      // Snapshot.withMutableSnapshot {
+        // elapsed++
+      // }
```

Comment:
note to self: fix once mosaic supports kt2.0

Reply:
I believe Mosaic `0.13.0` supports K2 now: https://github.com/JakeWharton/mosaic/releases/tag/0.13.0

Reply:
Nice, I can update to that; what I really mean is https://github.com/JakeWharton/mosaic/issues/340